Source/WebKit/UIProcess/API/C/playstation/WKView.cpp

```diff
@@ -56,7 +56,20 @@ WKCursorType toWKCursorType(const WebCore::Cursor& cursor)
 
 WKViewRef WKViewCreate(WKPageConfigurationRef configuration)
 {
+#if USE(WPE_BACKEND_PLAYSTATION)
+    RELEASE_ASSERT_WITH_MESSAGE(false, "API unavailable with WPE Backend PlayStation");
```

Comment:
It's nonsense to expose this API if this API just crashes.
Why don't you make this API fail?

Reply:
Or, not exposing this API if `USE(WPE_BACKEND_PLAYSTATION)`.

Reply:
I didn't see any `ENABLE` or `USE` in the WK API headers since those are internal definitions. That's why I made the API that.

In this one if there's no backend it'll crash later on so better to crash with a warning.

The long term goal is for this API to be deprecated and then replaced with the WPE one.

Reply:
Got it.